frontend/templates/views/partials/content.j2

```diff
@@ -36,6 +36,23 @@
   {% include '/views/partials/level-toggle.j2' %}
   {% endif %}
 
+  {# Render widget to toggle experiments if there are any#}
+  {% if doc.experimental %}
+  <div class="ap-m-experiments">
+    <h4>Experimental</h4>
+    <p>{{ _('This page could contain example previews with experimental features. Enable the experiment via the button below. Some components require the AMP Dev Channel to be enabled as well.') }} Learn more about <a href="{{ g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/experimental.md', locale=doc.locale).url.path }}">experimental features</a>.</p>
```

Comment:
Can we show this only when there are actual samples on the page?

Reply:
I don't think so, because the samples are written down as shortcodes in the content files. So we don't have any indicator, that says us if any or how many `[example ...]` shortcodes are in the content. That's why I adjusted the text.

Do you have an idea?

Reply:
The ref doc importer could add a flag if the page contains interactive samples. @matthiasrohmer wdyt?